### DISCUSSION TITLE
Hydration failed because the initial UI does not match what was rendered on the server

### DISCUSSION THREAD

**Discussion by Ebibeck** [vercel/next.js#47707] [Help]
Hydration failed because the initial UI does not match what was rendered on the server

**Comment by icyJoseph** [comment]
Got more data? some screenshots? stack trace? pseudo-code?

If you disable JavaScript and load your page, what do you see? vs seeing it with JavaScript enabled?

**Reply by Ebibeck** [reply]
Ok on it

**Reply by Ebibeck** [reply]
![Uploading IMG_20230330_194627_607.jpg …]()

![Uploading IMG_20230330_194258_514.jpg …]()

![IMG_20230330_194650_657.jpg](https://user-images.githubusercontent.com/92293938/228934935-19d77e19-aabc-410a-8cc1-6f78b62fbabb.jpg)

![Uploading IMG_20230330_194709_360.jpg …]()

**Reply by icyJoseph** [reply]
Not all images loaded properly. Wait for them to uploaded before submitting the response, Github's like that... 🤷 

I can see that this is within appDir. Could it be that some Date/timestamp is not correct?

**Reply by Ebibeck** [reply]
I'm not sure because initially it worked but after I refreshed it enters a infinite loop
